### ISSUE TITLE
Reformat the code with best practices using TSLint

### ISSUE THREAD

**Issue by Leoat12** [Leoat12/toread-cli#8]
It is always important to have a clean and easy to understand code and in my opinion it is done by using best practices recognized by the community, so my idea is using TSLint to reformat the code based on the its suggestions, which  already proved to be good practices. 
I already added TSLint to the dev dependencies and I'm working now on reformatting the code.  

It is under discussion, though, what rules will be used, so use this issue to discuss changes on the `tslint.json` file. For example, I added a exception to the recommended style by putting `"no-console": false` since we use `console.info()` and `console.error()` and others a lot. It may change if we find better ways to print on the screen.

**Comment by athif23** [comment]
I just found some tool named [signale](https://github.com/klauscfhq/signale) that can be used to print on the screen.

What do you think?

**Comment by Leoat12** [comment]
I was aware of `signale` when I studied `taskbook` code, but we also need to think about the amount of dependencies we already have and if it is worth adding another one. `signale` has much more features that we can use to beautify the CLI interactiveness. However, if we just want to avoid `console.info()`, it is better use `process.stdout.write()`, which is native from nodejs. 
To avoid writing this long call every time we can do this on `Display`:
```typescript
private static write(message: string){
process.stdout.write(message);
}
```
What do you think, use `signale` to refactor the display of the CLI entirely using it or just change `console.info()` for `process.stdout.write()`?
I will study `signale` a little bit more meanwhile.

**Comment by athif23** [comment]
I agree, let's not use `signale`, the dependencies we have already too much.
But,  I just want to know what the advantage of using `process.stdout.write()` than `console.info()`?

**Comment by Leoat12** [comment]
[TSLint](https://palantir.github.io/tslint/rules/no-console/) says that `console` methods are not suitable for production. I think it is true for web applications, but not for NodeJS applications, right? Maybe we can stay with `console.info()` until further notice. Don't you agree?
